static/app/views/profiling/landing/slowestFunctionsTable.tsx

```diff
@@ -0,0 +1,297 @@
+                  <SlowestFunctionCell>{t('p95()')}</SlowestFunctionCell>
+                  <SlowestFunctionCell>{t('p99()')}</SlowestFunctionCell>
+                  <SlowestFunctionCell>{t('Sum()')}</SlowestFunctionCell>
+                </SlowestFunctionHeader>
+                {sortedMetrics.slice(pagination.start, pagination.end).map((f, i) => {
+                  return (
+                    <SlowestFunction
+                      key={i}
+                      function={f}
+                      projectsLookupTable={projectsLookupTable}
+                    />
+                  );
+                })}
+              </SlowestFunctionsContainer>
+            </Fragment>
+          )}
+        </ContentContainer>
+      </SlowestWidgetContainer>
+      <SlowestFunctionsPaginationContainer>
+        <ButtonBar merged>
+          <Button
+            icon={<IconChevron direction="left" />}
+            aria-label={t('Previous')}
+            size={'sm'}
+            {...pagination.previousButtonProps}
+          />
+          <Button
+            icon={<IconChevron direction="right" />}
+            aria-label={t('Next')}
+            size={'sm'}
+            {...pagination.nextButtonProps}
+          />
+        </ButtonBar>
+      </SlowestFunctionsPaginationContainer>
+    </Fragment>
+  );
+}
+
+interface SlowestFunctionProps {
+  function: NonNullable<Profiling.Schema['metrics']>[0];
+  projectsLookupTable: Record<string, Project>;
+}
+
+function SlowestFunction(props: SlowestFunctionProps) {
+  const organization = useOrganization();
+
+  const example = props.function.examples[0];
+  const exampleLink =
+    example && typeof example !== 'string' && 'project_id' in example
+      ? generateProfileRouteFromProfileReference({
+          frameName: props.function.name,
+          framePackage: props.function.package,
+          orgSlug: organization.slug,
+          projectSlug: props.projectsLookupTable[example.project_id]?.slug ?? '',
+          reference: props.function.examples[0],
+        })
+      : null;
+
+  return (
+    <SlowestFunctionContainer>
+      <SlowestFunctionCell>
+        <Tooltip title={props.function.name}>
+          {exampleLink ? (
+            <Link to={exampleLink}>{props.function.name || t('<unknown function>')}</Link>
+          ) : (
+            props.function.name || t('<unknown function>')
+          )}
+        </Tooltip>
+      </SlowestFunctionCell>
+      <SlowestFunctionCell>
+        <Tooltip title={props.function.package || t('<unknown package>')}>
+          {props.function.package}
+        </Tooltip>
+      </SlowestFunctionCell>
+      <SlowestFunctionCell>
+        <SlowestFunctionsProjectBadge
+          examples={props.function.examples}
+          projectsLookupTable={props.projectsLookupTable}
+        />{' '}
+      </SlowestFunctionCell>
+      <SlowestFunctionCell>
+        {formatAbbreviatedNumber(props.function.count)}
```

Comment:
I'm still on the fence about exposing this because increasing the date range will **not** actually load more samples.

Reply:
I think we can remove it before exposing it to users, right now its just useful to look at from a dev pov.

Reply:
I'm going to add a todo